client/src/components/PropertiesMap.tsx

```diff
@@ -266,11 +266,13 @@ export default class PropertiesMap extends Component<Props, State> {
                 left: "10px",
               }}
             />
-            {useNewPortfolioMethod ? (
-              <BigPortfolioWarning sizeOfPortfolio={this.state.addrsPoints.length} />
-            ) : (
-              <></>
-            )}
+            <div className="MapBanner__container">
+              {useNewPortfolioMethod ? (
```

Comment:
small: could be `{useNewPortfolioMethod && <BigPortfolioBanner ... />}` to avoid the empty fragment

Reply:
I tried this but then the `false` comes through and it causes and error in the containing Map component

Reply:
ohh i see, the Map is doing something with the children. no worries then!